base/assets/js/index.js

```diff
@@ -0,0 +1,23 @@
+$(function(){
+
+  function toggleNav(){
+    const $nav = $("nav");
```

Comment:
パーツとして見なせるものは全体的にクラス名つけて、操作した方が良い気がしますが、
とりあえずは、以下のようにした方がよくなるかもです。

```suggestion
    const $nav = $ ("header").find("nav");
```

Reply:
修正しました
3516f99